src/reducers/people.js

```diff
@@ -130,6 +131,11 @@ const currentUserReducer = (state = null, action) => {
     case FETCH_LIVE_STATUS:
       const { new_notification_count } = payload
       return {...state, new_notification_count}
+    case INCREMENT_UNREAD_THREADS:
```

Comment:
can you merge my recent changes into this branch? i moved currentUserReducer into a separate file